content/recipes/migration.mdx

```diff
@@ -48,19 +48,19 @@ function useStores() {
 }
 ```
 
-That's all the basics you need to understand. You can simply use such hook for any function components you have refactored to hooks.
+That's all the basics you need to understand. You can simply use such a hook for any function components you have refactored to hooks.
 
-You can build on top of that and make a solution that fits your needs. For example have a hook for each separate store, eg. `useUserStore` or `useOrderStore`. It's a similar to passing string to inject, but much safer.
+You can build on top of that and make a solution that fits your needs. For example, have a hook for each separate store: e.g., `useUserStore` or `useOrderStore`. It's similar to passing string to `inject`, but much safer.
 
-### Why the hook is not exported from package?
+### Why the hook is not exported from the package?
 
-It's becoming a good practice for packages to always hide the Context object and only export a relevant hook. Usually the reason are to abstract some internals tucked away in context. However, in case of MobX, there is nothing to hide, it's stores you have added there.
+It's becoming a good practice for packages to always hide the Context object and only export a relevant hook. Usually the reasons are to abstract some internals tucked away in context. However, in case of MobX, there is nothing to hide, it's stores you have added there.
```

Comment:
I think I actually want to drop the whole paragraph, it's not that relevant and more confusing.
```suggestion
```

Reply:
You need to click on Commit suggestion since you haven't checked "Allow edit for maintainers".